### ISSUE TITLE
Fails to copy files to microSD card

### ISSUE THREAD

**Issue by wifon** [hanwen/go-mtpfs#94]
I've just noticed it's not possible to copy large files to microSD card. I wanted to move 800MB video file, tried 3 times and failed all 3 times. One of three logs below: (all 3 were the same when it comes to errors)

> anon-kun in ~ 2015/05/10 20:16:54 FUSE mounted
> 2015/05/10 20:17:31 32563200 bytes in 21.640387233s: 1 mb/s
> 2015/05/10 20:17:34 454656 bytes in 515.359171ms: 0 mb/s
> 2015/05/10 20:18:50 fatal error LIBUSB_ERROR_TIMEOUT; closing connection.
> 2015/05/10 20:18:50 AndroidSendPartialObject failed: mtp: cannot run operation ANDROID_SEND_PARTIAL_OBJECT, device is not open
> 2015/05/10 20:18:50 AndroidSendPartialObject failed: mtp: cannot run operation ANDROID_SEND_PARTIAL_OBJECT, device is not open
> 2015/05/10 20:18:50 111583232 bytes in 1m14.701116526s: 1 mb/s
> 2015/05/10 20:18:50 AndroidEndEditObject failed: mtp: cannot run operation ANDROID_END_EDIT_OBJECT, device is not open
> 2015/05/10 20:18:53 GetStorageInfo 20001: mtp: cannot run operation GetStorageInfo, device is not open
> 
> [1]  + done

Then I tried to copy said file to internal storage and there were absolutely no problems.
I run Arch Linux, go-mtpfs from AUR (git version), my device is Motorola Moto G 2nd

**Comment by hanwen** [comment]
does it work if you disable the android extensions?

**Comment by wifon** [comment]
No luck. Now I have LIBUSB_ERROR_TIMEOUTs and "cannot close file" error message from my FM (Thunar).
I can fusermount -u, mount again and it works for few files, then again the error

Now that I looked at this a bit more, it doesn't matter if I copy to internal memory or SD card, it doesn't matter what the file size is as well. Tried to copy few hundred ~1,5-2MB jpg files and it failed too. The difference is some of this jpgs were actually copied and if it fails on a big file, you won't be able to play it..
